store/entity-store.html

```diff
@@ -75,6 +76,21 @@
 			this._invalidationListeners.delete(listener);
 		},
 
+		_getToken: function(token) {
+			const tokenPromise = (typeof (token) === 'function')
+				? token()
+				: Promise.resolve(token);
+
+			return tokenPromise.then( tokenValue => {
+				const { sub, tenantid, scope, actualsub = 'actual' } = JSON.parse(atob(tokenValue.split('.')[1]).toString())
```

Comment:
I don't love this line. I'm open to other ideas. We could use `jwtDecode` or some other library. This all becomes easier when we convert to Polymer 3 since adding `node_modules` gets a lot nicer.

Reply:
Could you do this like the code it was sourced from? Blacklisting volatile claims instead of whitelisting? This risks cache collisions if a new claim is added

Reply:
It wasn't sourced from anything but I think I found the method you are referring to so I'll try to do it in a similar way.